batch/scripts/create_vector_tile_cache.sh

```diff
@@ -10,31 +10,54 @@ set -e
 # -z | --max_zoom
 # -t | --tile_strategy
 # -I | --implementation
+
+# optional arguments
+# --filter
+
 ME=$(basename "$0")
 . get_arguments.sh "$@"
 
-# Set TILE_STRATEGY
+
+NDJSON_FILE="data.json"
+
+echo "Fetching NDJSON data from the Data Lake: ${SRC} -> ${NDJSON_FILE}..."
+aws s3 cp "${SRC}" "${NDJSON_FILE}" --no-progress
```

Comment:
Minor nit:  is there a reason you moved the copy here earlier in the script.  If not, I would move it back to where it was, since you don't want to do the copy if we fail in the case statement below with a bad tile cache option.  (I know that is not likely to happen, because other code is generating this command.)

Reply:
That's a good point, the reason for the move is lost to the sands of git. I will move it back, thanks!